src/olympia/migrations/1043-remove-thunderbird-seamonkey-categories.sql

```diff
@@ -0,0 +1,8 @@
+-- Removing those categories and associated m2m will hide them from the site,
+-- and prevent developers from choosing them when submitting new add-ons or
+-- editing details for an existing one.
+DELETE addons_categories FROM addons_categories
+    INNER JOIN categories
+    ON addons_categories.category_id=categories.id
+    WHERE categories.application_id IN (18, 59);
+DELETE FROM categories WHERE application_id IN (18, 59);
```

Comment:
we can get rid of 52 and 60 too (sunbird and xul android).  Or maybe it should be `NOT IN (1, 61)`?

Reply:
Or... can we?  What happens if there are non-deleted addons using those categories?  (do we still have any sunbird or ancient android supporting add-ons?)

Reply:
Even though obvious when looking at git log it may help to write down which categories are 18, 59 etc. Simply add their names to the comment may make this much easier to grasp later.

Reply:
I will, but note that it's right there in the name of the file :p

Reply:
noted, I'm blind :eyes:

Reply:
Rebased into 520898d2fa503c932ae7c1b898875d88033aa890 which takes care of both comments.